src/main/java/no/neic/tryggve/constants/ConfigName.java

```diff
@@ -3,7 +3,6 @@
 public final class ConfigName {
     public static final String HOST = "host";
     public static final String HTTP_VERTICLE_PORT = "http_verticle_port";
-    public static final String SSL = "ssl";
```

Comment:
This is used to determine if the service is using ssl or not. Right now, it is not useful any more.  We config the ssl in nginx directly.